Documentation/ceph-object-bucket-claim.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Object Bucket Claim CRD
+weight: 2800
+indent: true
+---
+
+# Ceph Object Bucket Claim
+
+Rook supports the creation of new buckets (and access to existing buckets) via two custom resources:
+
+- an `Object Bucket Claim (OBC)` is custom resource which requests a bucket (new or existing) and is described by a Custom Resource Definition (CRD) shown below.
+- an `Object Bucket (OB)` is a custom resource generated when a bucket is provisioned and is described by an OB CRD, also shown below.
+
+An OBC references a storage class which is created by an administrator. The storage class defines whether the bucket requested is a new bucket or an existing bucket. It also defined the bucket retention policy.
+
+## Sample
+##### Storage class
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: rook-ceph-bucket
+provisioner: ceph.rook.io/bucket
+reclaimPolicy: Delete
+parameters:
+  objectStoreName: my-store
+  objectStoreNamespace: rook-ceph
+  region: us-east-1
+  #bucketName:
+```
+##### Object Bucket Claim
+```yamlapiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: ceph-bucket
+spec:
+  #bucketName:
+  generateBucketName: ceph-bkt
+  storageClassName: rook-ceph-bucket
+```
+
+## Object Bucket Claim Settings
+
+### Storage Class
+
+##### Metadata
+
+- `name`: The name of the storage class need to create, which will be reflected in the object bucket claim.
```

Comment:
Change to "_`name`: The name of the storage class which will be referenced by the object bucket claim._" -- to fix grammar.